src/librustdoc/clean/cfg.rs

```diff
@@ -389,98 +390,84 @@ fn write_with_opt_paren<T: fmt::Display>(
     Ok(())
 }
 
+impl Display<'_> {
+    fn display_quantified_sub_cfgs(
```

Comment:
I really don't think this is a great name for this function, but couldn't come up with a better one 😅

Reply:
I would just go for `display_sub_cfgs` since the only constructs (of the `cfg` sublanguage) that may have sub-cfgs are `all` and `any`.

I guess “quantified” is not wrong since it's true that the sub cfgs are quantified by either $\forall$ or $\exists$ over a finite domain $\mathbb{F}_n$ aka "n-ary and" ($\bigwedge\_{i=0}^n$) and "n-ary or" ($\bigvee\_{i=0}^n$), respectively. Anyway ^^

Reply:
Renamed. I just looked up "common term for ALL and ANY" or something haha. Math is not my strong suit, so thanks for the thorough explanation, I appreciate it 😁

Reply:
I was considering making a `Quantifier` enum with `Any` or `All` variants to make this function a bit easier to understand, but not sure if it's not too verbose. Leaving at is as is for now, unless you think it's a good idea.